README.md

```diff
@@ -12,16 +12,54 @@ coordination between nodes is necessary, this is where the
 become interesting and it is worth consider making the nodes form an Akka
 Cluster.
 
+## Kubernetes Instructions
+    
+### Docker Desktop for Kubernetes
+If you use Kubernetes on Docker Desktop, after turning it on, you should first issue:
```

Comment:
Perhaps link to https://www.docker.com/products/docker-desktop and clarify that this only applies to Mac and Windows users?